### DISCUSSION TITLE
Ability to change required fields and add order meta/data

### DISCUSSION THREAD

**Discussion by maggsy78** [woocommerce/woocommerce#45086] [WooCommerce, Blocks, and Site Editing]
## What do you like about the Cart & Checkout blocks?
Easy to style, much nicer looking than default layouts

## What do you think is missing from the Cart & Checkout blocks?
Ability to change required fields and add order meta/data

**Comment by nielslange** [comment]
Thanks for this suggestion, @maggsy78! I've added this suggestion to the discussion board, so that others can vote for this issue.
